### ISSUE TITLE
Allow hotlinking to Mumble during game

### ISSUE THREAD

**Issue by StarGate01** [StarGate01/AmongUs-Mumble#27]
**Is your feature request related to a problem? Please describe.**
Users can't link Mumble when the game is already running, as the initialization is only performed once at game start. 

**Describe the solution you'd like**
If Mumble is not linked, periodically check if it can be linked. 

**Additional context**
We can use a mechanism similar to the heartbeat loop.

**Comment by alisenai** [comment]
As per 83436af272f9f4330fb65b6af3358b1db7222a47, this is now implemented.

@StarGate01 does this include Mumble closing/restarting mid-game? If not, feel free to close the issue.

**Comment by StarGate01** [comment]
Thank you!

I moved the reconnection to a background thread, see 8fda5315f28db784aa2ff967810d37c53696814e. The following works now:
 - Reconnect to Mumble if mumble restarts while the game is running, this is due to the shared memory file being kept open by the game and Mumble just picks it up a start instead of creating it.
 - Wait for Mumble to start
 - If the game restarts while Mumble is running, it just reconnects as before.

I noticed that Mumble sometimes needs up to 10 seconds to realize a link has been established, but there is nothing we can really do about that. Maybe add it to the FAQ so users are patient.
